proposals/cloudedgeclientCG/charter.md

```diff
@@ -9,15 +9,15 @@
 
 ## Goals
 
-The mission of this group is to explore mechanisms and interfaces between Central Cloud, Edge-Cloud, and Client for computing workload offloading and orchestration. 
+The mission of this group is to explore mechanisms and interfaces between Cloud, Edge, and Client for computing workload offloading and orchestration. 
 The typical use cases include AI acceleration, cloud gaming, and streaming acceleration. The goal is to explore the feasibility of defining a set of new APIs and mechanisms 
-that enable computing workload offloading and orchestration between Central Cloud, Edge Cloud, and Client. It should leverage existing mechanisms as much as possible 
+that enable computing workload offloading and orchestration between Cloud, Cloud, and Client. It should leverage existing mechanisms as much as possible 
```

Comment:
```suggestion
that enable computing workload offloading and orchestration between Cloud, Edge, and Client. It should leverage existing mechanisms as much as possible 
```

Reply:
suggestion merged.

Reply:
Treating my suggestion as a comment, won't let me resolve it myself, github config issue - just ignore this, problem has been fixed.